rust/README.md

```diff
@@ -0,0 +1,42 @@
+# Rust on CodeQL
+
+## Development
+
+### Dependencies
+
+At present you need to be in the `rust-experiment` branch (or a branch on top of that) in both the `semmle-code` and `codeql` repos.
+
+TODO: are there any dependencies that aren't already CodeQL dependencies?  bazel and rustc?
+
+### Building the Rust Extractor
+
+This approach uses a released `codeql` version and is simpler to use for QL development. From your `semmle-code` directory run:
+```bash
+bazel run @codeql//rust:rust-installer
+```
+You now need to create a [per-user CodeQL configuration file](https://docs.github.com/en/code-security/codeql-cli/using-the-advanced-functionality-of-the-codeql-cli/specifying-command-options-in-a-codeql-configuration-file#using-a-codeql-configuration-file) and specify the option:
+```
+--search-path Users/YOURUSERNAME/semmle-code/ql
```

Comment:
I'm guessing this does not need an absolute path and gets joined with the home directory?

Reply:
It's a bit fussy.  `--search-path ./ql` works, I have a distaste for using the "current working directory" in a context like this where I'm not in control of it, but if someone can promise it will always be `semmle-code` then I'm OK with it.  `--search-path ~/semmle-code/ql` and a few other variations of that I tried do not work.

Reply:
yeah, I think I always used full absolute paths. Maybe I'd just put a generic `/path/to/semmle-code/ql`? `Users` is macos only, and in any case my local checkout for example is at `/workspaces/semmle-code` in codespace style

Reply:
Updated.